src/Avalonia.Visuals/Rendering/SceneGraph/ExperimentalAcrylicNode.cs

```diff
@@ -44,7 +43,7 @@ internal class ExperimentalAcrylicNode : DrawOperation
         /// Determines if this draw operation equals another.
         /// </summary>
         /// <param name="transform">The transform of the other draw operation.</param>
-        /// <param name="brush">The fill of the other draw operation.</param>
+        /// <param name="material">The fill of the other draw operation.</param>
```

Comment:
@danwalmsley is this comment still correct when for the `material` parameter on this class? If so, we can use it for the missing parameter comment above.